src/lighteval/models/base_model.py

```diff
@@ -267,8 +267,11 @@ def _init_max_length(self, max_length) -> int:
             if hasattr(self._config, attr):
                 return getattr(self._config, attr)
 
-        if hasattr(self.tokenizer, "model_max_length"):
-            return self.tokenizer.model_max_length
+        try:
+            if hasattr(self.tokenizer, "model_max_length"):
+                return self.tokenizer.model_max_length
+        except AttributeError:
+            hlog("No max length config setting is found in the model or tokenizer. max_length set to 2048.")
```

Comment:
interesting. this functions need tokenizer to be defined but the setup tokenizer function needs `max_length` to be defined as well. this will fail if we cannot find the sequence length in the model config. that means that we can simply remove this whole try catch (it will always fail) and just return 2048.

Reply:
Hi, thanks for the review. Should I just delete the whole try-except and update this branch accordingly?

Reply:
hi ! yes as this will always fail

Reply:
Hi, I've just deleted the relevant lines and updated the branch:)